content/oci/oci.go

```diff
@@ -52,12 +52,20 @@ type Store struct {
 	//     to manually call SaveIndex() when needed.
 	//   - Default value: true.
 	AutoSaveIndex bool
-	root          string
-	indexPath     string
-	index         *ocispec.Index
-	storage       *Storage
-	tagResolver   *resolver.Memory
-	graph         *graph.Memory
+
+	// AutoGC controls if the OCI store will automatically clean newly produced
```

Comment:
Just curious, what does **newly produced** mean?

Reply:
It means the new dangling nodes produced by this operation of `Delete()`.